### DISCUSSION TITLE
Help: New Meeting Note

### DISCUSSION THREAD

**Discussion by ms3056** [chhoumann/quickadd#537] [Q&A]
I currently track my meetings and link to their notes on my Daily Note. It looks something like this:

```markdown
# Meetings
- [ ] `08:00` **meeting name** [Notes](Journal/Notes/2023-Jul/20230719093214)
- [ ] `10:00` **meeting name** [Notes](Journal/Notes/2023-Jul/20230719093219)
```

I notice the filename is a unique identifier down to the second. I would like the following workflow:

1. Capture the meeting time in HH:MM format - so say 10:00 or 15:30 - I would type this
2. Capture the meeting name - this would be `meeting name` above
3. Create an entry on my daily note under the meetings heading like what you see above
4. Create the meeting note and open it with my meeting template, with the exact same name as you see in the link

My problem is I am wondering can I capture the unique filename string and then reuse it so that the entry in the Daily Note matches the filename creation?

**Comment by chhoumann** [comment]
Really sorry this one sat unanswered for so long.

Everything you wanted is doable natively now, and the tricky part - reusing the exact generated filename in both the meeting note and the daily-note link - has a clean solution. `{{FILENAMECURRENT}}` resolves to the note QuickAdd just created and opened, so you never have to evaluate `{{DATE}}` twice and risk the two landing a second apart.

Here's the whole thing. Create a **Macro** choice ("New meeting") with two steps:

**Step 1 - Template** (creates the meeting note):
- Template path: your meeting template
- File Name Format: `{{DATE:YYYYMMDDHHmmss}}` (unique to the second)
- Open the created note: **on**
- In the template body, use `{{VALUE:time}}` and `{{VALUE:meeting name}}` - you'll be prompted for them here, once.

**Step 2 - Capture** (adds the checklist line to your daily note):
- Capture To: your daily note
- Insert after: `# Meetings`
- Capture format:

```
- [ ] `{{VALUE:time}}` **{{VALUE:meeting name}}** [[{{FILENAMECURRENT}}]]
```

Because both steps run in the same pass, `{{VALUE:time}}` and `{{VALUE:meeting name}}` are asked only once (in step 1) and reused in step 2, and `{{FILENAMECURRENT}}` points at the meeting note step 1 just made - so the link in the daily note matches the filename exactly.

Entering "Standup" and "14:30" created `20260706191957.md` from the template and dropped `- [ ] `14:30` **Standup** [[20260706191957]]` under `# Meetings` in the daily note, with the values only prompted once.

If you'd rather use a Markdown-style link, `{{FILENAMECURRENT}}` still gives you the name to build one, e.g. `[{{VALUE:meeting name}}]({{FILENAMECURRENT}}.md)`.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#filenamecurrent
